R/sgdnet.R

```diff
@@ -0,0 +1,103 @@
+#' Fit a Generalized Linear Model with Elasticnet Regularization
```

Comment:
Elastic Net is two words

Reply:
I adopted this from the glmnet documentation, but I agree.